packages/react-dev-utils/InterpolateHtmlPlugin.js

```diff
@@ -15,7 +15,6 @@
 // Learn more about creating plugins like this:
 // https://github.com/ampedandwired/html-webpack-plugin#events
 
-'use strict';
```

Comment:
We could override extending our own config, but I'm not sure if that's necessary.